src/js_native_api_v8.cc

```diff
@@ -1718,6 +1718,55 @@ napi_status NAPI_CDECL node_api_create_property_key_utf16(napi_env env,
   });
 }
 
+napi_status node_api_create_property_key_utf8(napi_env env,
+                                              const char* utf8name,
+                                              size_t length,
+                                              napi_value* result) {
+  if (env == nullptr || utf8name == nullptr || result == nullptr) {
+    return napi_invalid_arg;
+  }
+
+  napi_status status = v8impl::NewString(
```

Comment:
We should use `return` here as in the `node_api_create_property_key_utf16`. Checking the return status does not bring any value. In the end we just return what the `v8impl::NewString` returned anyway.